.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Added to support https://github.com/GoogleChrome/lighthouse/pull/14977/files#r1199641414